client/components/fields/preview/base/converters.ts

```diff
@@ -53,7 +53,7 @@ export function getG2ContentTypeString(
 
 export function getValuesFromCV(
     propName: string,
-    valueField: string = 'qcode',
+    valueField: string = propName === 'agendas' ? '_id' : 'qcode',
```

Comment:
Rather than adding a specific default `valueField` here based on the `propName`, it would be easier to maintain if the `valueField` was provided where it is used.
https://github.com/superdesk/superdesk-planning/blob/develop/client/components/fields/preview/index.ts#L40